starter_code/views/index.hbs

```diff
@@ -1,3 +1,5 @@
-<div class="index-container">
-  
+<div id="main" class="index-container">
+    <img id="imagebeer" src="/images/beer.png" alt="beer">
+    <button> Check the Beers!!!</button>
```

Comment:
You should have used a tags here - you need to link to the beers or random-beer page